### PR TITLE
PySWMM v1.0

### LINKED ISSUE #263: Deprecate DLL selector function
PySWMM v1.0 refactors swmm5 module to use [swmm-toolkit](https://github.com/OpenWaterAnalytics/swmm-python) to interact with model data from [SWMM API](https://github.com/OpenWaterAnalytics/Stormwater-Management-Model). 

Due to this change, the ability to select different .dll, dylib, and .so is removed after version 1.0.

### LINKED ISSUE #264: Refactor PySWMM to use swmm-toolkit package
Refactor pyswmm swmm5 module to use [swmm-toolkit](https://github.com/OpenWaterAnalytics/swmm-python) package to interact with model data from [SWMM API](https://github.com/OpenWaterAnalytics/Stormwater-Management-Model) instead of using ctypes and .dll.

### LINKED ISSUE #265: Repository clean up
Clean up PySWMM repository by deleting unused files:
- travisCI.yml (not necessary with swmm-toolkit replacement)
- .ciocheck
- .coveragerc
- setup.cfg
- .dll files and references
- requirements-test.txt
- requirements-travis.txt

### PR DESCRIPTION
**Pull request by jennwuu**
This pull request refactors pyswmm swmm5 module to use [swmm-toolkit](https://github.com/OpenWaterAnalytics/swmm-python) package to interact with model data from [SWMM API](https://github.com/OpenWaterAnalytics/Stormwater-Management-Model). Moving forward, PySWMM is only compatible with Python 3+ 64bit. 

Setup Github Action
Closes #263 The ability to use different SWMM .dll is also removed. 
Closes #264 swmm5 module refactor
Closes #265 Repository cleanup

### CODE REVIEW

File: pyswmm/__init__.py [hunk 1]
```diff
@@ -17,7 +17,8 @@
 from pyswmm.system import SystemStats
 from pyswmm.raingages import RainGages, RainGage
 
-VERSION_INFO = (0, 6, 3, 'dev0')
+VERSION_INFO = (1, 0, 0, 'dev0')
```
Comment (bemcdonnell) [inline pyswmm/__init__.py:20]: @jennwuu can you confirm the versions here?

**Review by bemcdonnell — APPROVED** [review]

**Comment by bemcdonnell** [comment]
Please make sure to clearly note the deprecation of the DLL picker feature in the release notes

File: pyswmm/__init__.py (hunk 1 above)
Comment (jennwuu) [reply pyswmm/__init__.py:20]: @bemcdonnell the SWMM version?

Comment (jennwuu) [reply pyswmm/__init__.py:20]: @bemcdonnell I think these version information should removed since swmm is from swmm-toolkit package

**Review by michaeltryby — APPROVED** [review]
I approve these changes.

**Review by abhiramm7 — APPROVED** [review]
Looks great!

**Review by katmratliff — APPROVED** [review]
Nice work!

File: pyswmm/__init__.py (hunk 1 above)
Comment (jennwuu) [reply pyswmm/__init__.py:20]: @bemcdonnell *poke*

**Comment by michaeltryby** [comment]
🎈🥳🎉

**Comment by bemcdonnell** [comment]
💃🕺💃🕺💃🕺💃🕺💃💃

**Pull request merged**